### PR TITLE
adding ubooquity

### PR DESCRIPTION
**Pull request by jesse-troy**
Adding Ubooquity as requested from #148 

This one does require a little bit of specific configuration if it is to be exposed externally (and you want to simply use a subdomain). This is documented in the application page for this app.

Once this is merged in I will add Mylarr, which is a management app similar to Radarr/Sonarr, but for comics.

### CODE REVIEW

File: docs/applications/ubooquity.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Ubooquity Comic and Book Server
+
+Homepage: [https://vaemendis.net/ubooquity/](https://vaemendis.net/ubooquity/)
+Documentation: [https://vaemendis.github.io/ubooquity-doc/](https://vaemendis.github.io/ubooquity-doc/)
+Docker Image: [https://hub.docker.com/r/linuxserver/ubooquity/](https://hub.docker.com/r/linuxserver/ubooquity/)
+
+Ubooquity is a free, lightweight and easy-to-use home server for your comics and ebooks. Use it to access your files from anywhere, with a tablet, an e-reader, a phone or a computer.
+po
```
Comment (tylergarcianet) [inline docs/applications/ubooquity.md]: `po` Is this a typo?

File: tasks/ubooquity.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    env:
+      TZ: "{{ ansible_nas_timezone }}"
+      PUID: "{{ ubooquity_user_id|quote }}"
+      PGID: "{{ ubooquity_group_id|quote }}"
+      MAXMEM: "1024"
+    ports:
+      - "2202:2202"
+      - "2203:2203"
+    restart_policy: unless-stopped
+    #memory: 1g
```
Comment (tylergarcianet) [inline tasks/ubooquity.yml]: Is there a reason why the memory limit is commented out?

File: docs/applications/ubooquity.md (hunk 1 above)
Comment (jesse-troy) [reply docs/applications/ubooquity.md]: Yes. I will remove it.

File: tasks/ubooquity.yml (hunk 2 above)
Comment (jesse-troy) [reply tasks/ubooquity.yml]: If you take a look at the environment variables you will see that the application is limited via this variable. I have been running it exactly like this with no issues for nearly a year now. Woud you like for me to just remove the text on line 28?

**Comment by jesse-troy** [comment]
@tylergarcianet I have made changes to address the issues you pointed out. I don't think I have the ability to fix this new conflict. I did not see this conflict when I submitted this PR. Can you help me out or tell me what steps to take to get this merged?

**Comment by bcurran3** [comment]
@jesse-troy 

I've tested this and it works for me.

The conflict that I see is that you simply need to add a return at the end of your additions to all.yml.dist.

And thanks for this!

**Comment by davestephens** [comment]
I will get this merged soon guys, thanks for your patience.

Real life has taken over somewhat in the last six weeks, so hacking time is
limited!

**Comment by davestephens** [comment]
Done, and thanks :+1:

**Pull request merged**